src/Streamly/Data/String.hs

```diff
@@ -242,6 +247,15 @@ lines = foldLines A.write
 words :: (MonadIO m, IsStream t) => t m Char -> t m (Array Char)
 words = foldWords A.write
 
+-- | Unfold a stream to character streams using the supplied 'Unfold'
+-- and concat the results suffixing a newline character @\\n@ to each stream.
+--
+-- > unfoldLines = S.intercalateSuffix UF.singleton '\n'
+--
```

Comment:
An example here would be nice

Reply:
I know, but too many things to do, did not have time for that :-)